docs/querying/sql-functions.md

```diff
@@ -504,6 +504,22 @@ Returns the current timestamp in the connection's time zone.
 
 Rounds down a timestamp by a given time unit.
 
+## DECODE_BASE64_COMPLEX
+
+`DECODE_BASE64_COMPLEX(expr1, expr2)`
+
+**Function type:** [Scalar, other](sql-scalar.md#other-scalar-functions)
+
+Decodes complex expressions represented as literals, where `expr1` is a string literal with a valid [complex type name](sql-scalar.md#complex-type-expressions) and `expr2` is a base64-encoded string that contains a serialized value of the type defined in `expr1`.
```

Comment:
This part isn't super clear to me:
```
`expr1` is a string literal with a valid complex type name
```

If I'm reading it correctly, maybe something like:
```
`expr1` is a string literal whose value is one of the complex types, and `expr2` is...
```